src/docfx/lib/markdown/MarkdigUtility.cs

```diff
@@ -64,6 +64,19 @@ public static MonikerList GetZoneLevelMonikers(this MarkdownObject obj)
             return default;
         }
 
+        public static string? GetTabbedConceptual(this MarkdownObject obj)
+        {
+            foreach (var parent in obj.GetPathToRootInclusive())
+            {
+                if (parent is TabContentBlock content)
+                {
+                    var inLine = (LinkInline)content.TabTitle.Inline;
+                    return inLine.Url.Substring("#tab/".Length);
```

Comment:
This method seems to be retrieving the `Id` property parsed out by `TabGroupAggregator.ParseHeading` method, so:
- Change the method name to `GetTabId`
- Add `Id` property to `TabContentBlock` and set it to the result of `TabGroupAggregator.ParseHeading`.

Reply:
OK